pallets/ocex/rpc/Cargo.toml

```diff
@@ -11,7 +11,15 @@ sp-rpc = { workspace = true }
 sp-blockchain = { workspace = true }
 sp-api = { workspace = true }
 sp-runtime = { workspace = true }
+sp-core = { workspace = true }
 orderbook-primitives = { path = "../../../primitives/orderbook" }
 pallet-ocex-runtime-api = { path = "runtime-api" }
 serde_json = { workspace = true, default-features = true }
 polkadex-primitives = { workspace = true }
+sc-rpc-api = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.43" }
```

Comment:
Please use it from workspace ([here](https://github.com/Polkadex-Substrate/Polkadex/blob/9978d15b148215b5ceaf6f42fa281290c79b6abf/Cargo.toml#L162) it is already added)